proxy.go

```diff
@@ -291,7 +293,7 @@ func (p *Proxy) CreateMessageFromRequest(r *http.Request) (Event, error) {
 	event.Username = username
 	checksum.Type = "sha256"
 	event.Checksum = []interface{}{checksum}
-	log.Info("user ", event.Username, " uploaded file ", event.Filepath, " with checksum ", checksum.Value, " at ", time.Now())
+	log.Info("user ", event.Username, " with pilot ", claims["pilot"], " uploaded file ", event.Filepath, " with checksum ", checksum.Value, " at ", time.Now())
```

Comment:
If there is no `pilot` is the `claims["pilot"]` then empty or is it populated with the users name?

Reply:
pilot should be the actual person so in that case it should be the same as the  `sub`